tests/suites/test_suite_ssl.function

```diff
@@ -1347,6 +1403,140 @@ static int ssl_populate_session( mbedtls_ssl_session *session,
     return( 0 );
 }
 
+/*
+ * Perform data exchanging between \p ssl_1 and \p ssl_2 and check if the
+ * message was sent in the correct number of fragments.
+ *
+ * /p ssl_1 and /p ssl_2    Endpoints represented by mbedtls_ssl_context. Both
+ *                          of them must be initialized and connected beforehand.
+ * /p msg_len_1 and /p msg_len_2 specify the size of the message to send.
+ * /p expected_fragments_1 and /p expected_fragments_2 determine in how many
+ *                          fragments the message should be sent.
+ *      expected_fragments is 0: can be used for DTLS testing while the message
```

Comment:
I think that it would be good to explain in more detail what does this case mean. Does it mean that the message is not sent? Is it not fragmented at all? Does the underlying library return an error?

Reply:
I extended that description